config/both-project-scratch-def.json

```diff
@@ -1,38 +1,38 @@
 {
-  "orgName": "Test B2B and B2C Company",
-  "edition": "Developer",
-  "features": [
-    "Communities",
-    "B2BCommerce",
-    "B2BAuraTemplateEnabled",
-    "B2CCommerceGMV",
-    "PersonAccounts",
-    "OrderManagement",
-    "EnableSetPasswordInApi"
-  ],
-  "settings": {
-    "lightningExperienceSettings": {
-      "enableS1DesktopEnabled": true
-    },
-    "experienceBundleSettings": {
-      "enableExperienceBundleMetadata": true
-    },
-    "communitiesSettings": {
-      "enableNetworksEnabled": true
-    },
-    "orderManagementSettings": {
-      "enableOrderManagement": true
-    },
-    "orderSettings": {
-      "enableOrders": true,
-      "enableEnhancedCommerceOrders": true,
-      "enableOptionalPricebook": true
-    },
-    "commerceSettings": {
-      "commerceEnabled": true
-    },
-    "languageSettings": {
-      "enableTranslationWorkbench": true
+    "orgName": "Test B2B and B2C Company",
+    "edition": "Developer",
+    "features": [
+        "Communities",
+        "B2BCommerce",
+        "B2BAuraTemplateEnabled",
+        "B2CCommerceGMV",
+        "PersonAccounts",
+        "OrderManagement",
+        "EnableSetPasswordInApi"
+    ],
+    "settings": {
+        "lightningExperienceSettings": {
+            "enableS1DesktopEnabled": true
+        },
+        "experienceBundleSettings": {
+            "enableExperienceBundleMetadata": true
+        },
+        "communitiesSettings": {
+            "enableNetworksEnabled": true
+        },
+        "orderManagementSettings": {
+            "enableOrderManagement": true
+        },
+        "orderSettings": {
+            "enableOrders": true,
+            "enableEnhancedCommerceOrders": true,
+            "enableOptionalPricebook": true
+        },
+        "commerceSettings": {
+            "commerceEnabled": true
+        },
+        "languageSettings": {
+            "enableTranslationWorkbench": true
+        }
     }
-  }
```

Comment:
I think the linter in husky is changing this. There is no change in the file.

Reply:
please revert the linter changes. I think we all should have the same linter configured